### PR TITLE
ci: pass secrets to docker and packages workflows

### PR DESCRIPTION
**Pull request by id**
Fixes workflows triggered on push.

Reference runs:
- [Build and push docker images](https://github.com/emqx/emqx/actions/runs/5808912990)
- [Cross build packages](https://github.com/emqx/emqx/actions/runs/5809437291)

### CODE REVIEW

**Review by Rory-Z — APPROVED** [review]

File: build [hunk 1]
```diff
@@ -368,21 +368,81 @@ docker_cleanup() {
...
+       --build-arg PKG_VSN="${PKG_VSN}" \
+       --file "${EMQX_DOCKERFILE}" \
+       --label org.opencontainers.image.title="${PROFILE}" \
+       --label org.opencontainers.image.edition="${EDITION}" \
+       --label org.opencontainers.image.version="${PKG_VSN}" \
+       --label org.opencontainers.image.revision="${GIT_REVISION}" \
+       --label org.opencontainers.image.created="${ISO_8601_DATE}" \
+       --label org.opencontainers.image.source='https://github.com/emqx/emqx' \
+       --label org.opencontainers.image.url='https://github.com/emqx/emqx' \
+       --label org.opencontainers.image.documentation='https://www.emqx.io/docs/en/latest/' \
```
Comment (thalesmg) [inline build:422]: Minor note: EE docs live in [docs.emqx.com](https://docs.emqx.com/en/enterprise/v5.1/).

File: build [hunk 2]
```diff
@@ -368,21 +368,81 @@ docker_cleanup() {
...
+       --pull
+    )
+    if [ "${DOCKER_BUILD_NOCACHE:-false}" = true ]; then
+        DOCKER_BUILDX_ARGS+=(--no-cache)
+    fi
+    if [ "${SUFFIX}" = '-elixir' ]; then
+        DOCKER_BUILDX_ARGS+=(--label org.opencontainers.image.elixir.version="${EMQX_BUILDER_ELIXIR}")
+    fi
+    if [ "${DOCKER_LATEST:-false}" = true ]; then
+        DOCKER_BUILDX_ARGS+=(--tag "${DOCKER_REGISTRY}/${DOCKER_ORG}/${PROFILE}:latest")
```
Comment (thalesmg) [inline build:439]: If it's latest & elixir, it should be `latest-elixir`

File: build [hunk 3]
```diff
@@ -368,21 +368,81 @@ docker_cleanup() {
...
+    fi
+    if [ "${DOCKER_PLATFORMS:-default}" != 'default' ]; then
+        DOCKER_BUILDX_ARGS+=(--platform "${DOCKER_PLATFORMS}")
+    fi
+    if [ "${DOCKER_PUSH-false}" = true ]; then
```
Comment (thalesmg) [inline build:444]:
```suggestion
    if [ "${DOCKER_PUSH:-false}" = true ]; then
```

**Review by thalesmg — APPROVED** [review]
:rocket:

**Pull request merged**